monitor/sth_fetcher.go

```diff
@@ -55,6 +56,10 @@ var sthStats = &sthFetchStats{
 		Name: "sth_inconsistencies",
 		Help: "Count of times two CT log signed tree heads (STHs) could not be proved consistent",
 	}, []string{"uri", "type"}),
+	sthFetchTotal: promauto.NewCounterVec(prometheus.CounterOpts{
```

Comment:
Want to move this next to `sth_failures` so the relationship is more obvious?

Reply:
Done, thanks!